test/config.sh

```diff
@@ -214,5 +215,47 @@ collect_coverage() {
     mv cover.router.prof $(mktemp -u ./coverage/integration.XXXXXXXX) || true
 }
 
+################################
+# performance utils
+################################
```

Comment:
I can move this code to a different place, but my plan is to add more performance tests, so they need to be in a common place... `perf.sh` maybe?